docs/recipes/add-custom-chain.mdx

```diff
@@ -0,0 +1,142 @@
+---
+sidebar_position: 5
+title: Add a custom chain
+description: Learn how to add custom chains to your project.
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Add a custom chain
+
+This recipe demonstrates how to add a custom chain to your project. We'll use Base as an example, but you can apply this process to any other chain you want to add.
+Scaffold-ETH 2 uses [viem/chains](https://viem.sh/docs/chains/introduction) as a list of chains.
+Normally, Base already exists in viem/chains and [you can import it and use it](/deploying/deploy-nextjs-app#--targetnetworks), but we're going to add it manually to show you how to do it.
+
+:::info
+Scaffold-ETH 2 consists of two parts:
+
+- `packages/nextjs`: nextjs frontend
+- `packages/hardhat` or `packages/foundry`: hardhat or foundry to deploy smart contracts
+
+The frontend and the hardhat/foundry project use a different set of chains.
+You should add the chain to both the frontend and your hardhat/foundry config.
```

Comment:
```suggestion
You should add the chain to both the frontend and your hardhat/foundry config. Checkout [deploying your smart contract](/deploying/deploy-smart-contracts) section on how to deploy different chains.
``` 

what do you feel about this? This way we don't have to mention Step 3 in this page? Because we are already explaining same thing in /deploying/deploy-smart-contracts?

Reply:
I think this is a great suggestion, making the change right now...